docs/HISTORY.txt

```diff
@@ -5,7 +5,11 @@ Changelog
 3.5.5 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Template API breakage - 'content-title' -> 'tabbedview-content-title'
```

Comment:
It may not be obvious to everybody that this is a slot, which was renamed.
How about `- Template API breakage - renamed slot 'content-title' to 'tabbedview-content-title'` ?

Otherwise 👍